Copyedit designing a timeline tabletop simulator

charlottejmc opened the pull request:
I am preparing to merge my copyedits of designing-a-timeline-tabletop-simulator, represented by Issue #553

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

First copyedits

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Additional copyedits

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Copyedits

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Extended copyedits

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Final copyedit and change references to endnotes

charlottejmc commented:
Hello @copystar and @RolRodr,

I've now prepared the copyedits for this lesson. I'd be grateful if you could review the adjustments and confirm that you are happy for me to merge these. You can see the details of my edits under the [files changed](https://github.com/programminghistorian/ph-submissions/pull/600/files) tab!

I'd like to bring your attention to the comments I have attached to certain specific lines. You can respond to any of my suggestions via the comments below, or click _Resolve conversation_ if you're happy with them like this:

![image](https://github.com/programminghistorian/jekyll/assets/143802849/6a73acb9-215d-4fe1-8ab7-247921a3e6a4)

If you want to make edits, please work by navigating to the [files changed](https://github.com/programminghistorian/ph-submissions/pull/600/files) and clicking the three dots at the upper right of the file, then _Edit file_:

<img width="803" alt="image" src="https://github.com/programminghistorian/ph-submissions/assets/143802849/2baa2f87-22a9-43c3-9a1b-c70881769930">

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Designing a _Timeline_ for the Tabletop and Tabletop Simulator"
+title: "Designing a _Timeline_ card deck for tabletops and Tabletop Simulator"
```
Hi @copystar and @RolRodr, I've suggested a slight alteration to the lesson title. Does this make good sense to you both?
I'm also not sure we can use italics in the lesson title for _Timeline_. We'll see what it looks like once we stage the lesson for publication, at which point we might have to make that small change.

copystar replied:
I'm okay with this title change!

charlottejmc replied:
Another alternative is "Designing a deck of _Timeline_ cards for tabletops and Tabletop Simulator"

copystar replied:
I like this second suggestion better!  I like that it carries the original 'tabletops and tabletop simulator'

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -24,27 +24,23 @@ doi: XX.XXXXX/phen0000
 ## Introduction
 
-We no longer require students to memorize tables of historical events and their respective years of occurrence through the practice of rote learning, as was done in mid-nineteenth century America (Rosenberg). Setting aside the broader arguments for and against teaching chronology in the classroom (Hodkinson and Smith), this lesson is written from the position that it is useful to “help students d ...
+We no longer require students to memorize tables of historical events and their respective years of occurrence through the practice of rote learning, as was done in mid-nineteenth century America.[^1] Setting aside the broader arguments for and against teaching chronology in the classroom,[^2] this lesson assumes that it is useful to 'help students develop a rich body of knowledge in our content  ...
```
As you can see already from this first line, I've changed all the citations to endnotes, as all our lessons follow the Chicago Manual of Style, 17th edition.

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+_Timeline_ is a card game designed by Frédéric Henry, first published in 2012 and still available for purchase through the global game publisher [Asmodee](https://www.asmodee.ca/product/timeline-access/) and its subsidiary [Zygomatic](https://www.dobblegame.com/en/games/#category-timeline). _Timeline_ can be played with 2 to 8 players. As it is a very simple game to learn to play with a group, _T ...
 
-> **_Timeline_** is a card game where each card, on both sides, depicts a specific historical event, invention or discovery, with only one side displaying the year in which that event occurred. Players take turns placing a card from their hand in a row on the table. After placing the card, the player reveals the date on it. If the card was placed correctly with the date in chronological order wit ...
+> **_Timeline_** is a card game where each card depicts a specific historical event, invention or discovery, but only one side displays the year in which that event occurred. Players take turns placing a card from their hand in a row on the table. After placing the card, the player flips it to reveal the date on the back. If the card was placed in chronological order with all the other cards on t ...
```
I'm not certain whether this passage is a citation or not. Some of the phrasing is similar to a description of _Timeline_ gameplay I found on [BoardGameGeek](https://boardgamegeek.com/boardgame/128664/timeline). Is this where you found it originally?

copystar replied:
I checked and found that I have previously bookmarked the same link to BoardGameGeek, so while I cannot recall if I got the phrasing from the site, I'd rather err on the side of caution and add a citation if the language is close. Thank you!

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+In this example, nanDECK was directed to draw a black, rectangular border around each card, and add some dotted cut marks.
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-4.png" alt="In between the top window of instructions and the bottom window of the program's log is text that begins, IMAGE=\"range\", \"image file\", position x, position y, width, and then the word, height, in bold, followed by other variables. In the top window, the mouse cursor is resting on the 6th instruction lin ...
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-4.png" alt="In between the top window of instructions and the bottom window of the program's log is text that begins, IMAGE=\"range\", \"image file\", position x, position y, width, and then the word, height, in bold, followed by other variables. In the top window, the mouse cursor is resting on the 6th instruction lin ...
```
I started encountering issues with figures here, up to Figure 9 (Line 549). I think that, when some figures were switched around, the alt-text descriptions got a little scrambled. Could you please check that the alt-text matches the image and the caption? I think you may also have to verify that the images are in the right place. I'll make a comment like this one for each of subsequent figure.

copystar replied:
I see the same. The comment for Figure 4 should read, "A set of five upright rectangles, each surrounded by a thick black border and faint dotted lines that extend off the page. These are a series of five cards, each bearing an image and a caption. The first card bears an image of a house with the caption, "Duff Baby house is built". The second card features an image of a commemorative plaque on a structure of stones and the words, "Treaty 2, or the McKee Purchase, was signed." The middle card features a portrait of young man wearing a military suit and a cap with a large feather and the caption, "Shawnee chief and warrior Tecumseh is born in present-day Ohio." The fourth card features a map of a fort and the words, "Settlement of now-Windsor began when the Huron Mission was moved from Bois Blanc to the South Shore." The last card features an image of figures crossing a river in a canoe and the words, "Detroit is founded by Antoine Cadillac".

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+In the instruction window, it can sometimes be difficult to determine exactly what parameter the numbers or terms listed after each directive refer to. If you want to know what parameter a variable represents, you can hover your mouse over the variable and check the syntax reminder below: nanDECK will show the relevant parameter name in bold.
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-5.png" alt="A screenshot of a card with dimensions 472 x 768. The card has a thick black border. The top of the card features a black and white image of a brick church with a large wooden door and branches of a tree that is out of frame. Under the image is, in bold print, 1851. Under this date is the text, 'The corners ...
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-5.png" alt="A screenshot of a card with dimensions 472 x 768. The card has a thick black border. The top of the card features a black and white image of a brick church with a large wooden door and branches of a tree that is out of frame. Under the image is, in bold print, 1851. Under this date is the text, 'The corners ...
```
Here as well, the alt-text does not seem to match the image.

copystar replied:
Yes, this was accidentally submitted for Figure 4.

Alt-text should read, "In-between the top window of instructions and the bottom window of the program's log is text that begins, IMAGE="range", "image file", pos x, pos y, width, and then the word, height, in bold, followed by other variables. In the top window, the mouse cursor is resting on the 6th instruction line, on the number 60%, which we now understand as the image's height."

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+nanDECK can very usefully calculate the number of items listed under a designated field in a spreadsheet. It does this using an expression delimited by curly brakets. In this case, the expression {(IMAGES)} instructs nanDECK to caculate the number of items listed under the `IMAGE` field. Here, there are 58 images, so {(IMAGES)} returns `58`. Knowing this, you'll see that at line 6 of Figure 5 abo ...
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption= ...
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption= ...
```
Here as well, the alt-text does not seem to match the image. Actually, I'm also unsure whether this is the right image here?

copystar replied:
The alt-text for Figure 6. should be, "A screenshot of a card with dimensions 472 x 768. The card has a thick black border. The top of the card features a black and white image of a brick church with a large wooden door and branches of a tree that is out of frame. Under the image is, in bold print, 1851. Under this date is the text, "The cornerstone of Sandwich First Baptist Church is laid on land donated by the Crown".

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+Rather than use complicated scripting to ensure that the year only appears on one side of each printed card, we instead provide instructions to print everything that was explicited designated in the linked Excel spreadsheet. In the spreadsheet, the first 58 rows provide an image, a description, and a year, to be printed on the face of the card. The next 58 rows only provide an image and a descrip ...
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7: The simplest way to have _nanDECK_ generate a card side without the date being printed is to duplicate the informatio ...
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7. The simplest way for nanDECK to generate a card side without the date is to duplicate the information and remove the  ...
```
Here again, the alt-text does not seem to match the image.

copystar replied:
Figure 7 alt-text should read, "A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card. "

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption= ...
 
-To generate the card above, _nanDECK_ was directed to insert the image found at _Images/Sandwich_First.jpg_ at the 0,0 position and for the image to take up 100% of the width (if possible) and only 60% of the card’s height (if possible). By using the P flag, _nanDECK_ is asked to calculate where the image should be positioned while maintaining the original image’s proportions. Since there are a v ...
+To generate the card above, nanDECK was directed to insert the image found at `Images/Sandwich_First.jpg`, to place it at the 0,0 position, and make the image take up 100% of the card's width (if possible), but only 60% of the height (if possible). The `P` flag asks nanDECK to maintain the image’s original proportions. I opted to make all the images grayscale using the `G` tag, to give the cards  ...
```
I'm a little confused by the use of both 'flag' and 'tag' here – are they different things, or should one of them be changed to match the other?

copystar replied:
Both instances should be 'flag' as this is the language the developer uses in the manual.  Perhaps this text would be more clear if it referred back to Figure 5... like, "as illustrated in Figure 5, by selecting the checkbox beside 'P', the flag tells nanDECK to maintain the image's original proportions.

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
````diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+```
 
-In order to align the cards with years printed on them, it is only necessary to designate their range. In the case of my personal deck, there are 59 unique cards, with the fronts being generated from range 1-59 in my spreadsheet and the backs generated from range, 60-118.
+In order to align the card fronts and backs, designate the range you want to duplicate. In the case of my personal deck, there are 59 unique cards: the fronts will be generated from cards in the 1-59 range, and the backs from the cards in the 60-118 range.
````
I'm not sure I understand exactly why we are counting cards to 59 (and thus doubling to 118) here, as I believe the deck only contains 58. Perhaps this is a small error?

copystar replied:
I checked my code and it's not an error. I believe that nanDECK counts oddly because it knows to skip the first row as it contains field names.

charlottejmc replied:
If that's the case, perhaps we should add a sentence to that effect, as it's not very clear to me at the moment. Here you say 'In the case of my personal deck, there are 59 unique cards' – but this doesn't seem right! Am I correct in understanding that there are 58 unique cards and images, but the card #58 would be on row 59 of the spreadsheet? 

Line 330 says 'there are 58 images, so {(IMAGES)} returns `58`. Knowing this, you'll see that at line 6 of Figure 5 below, `IMAGE="1-{(IMAGES)}"` instructs nanDECK to print the images from rows 1 through 58.'

 Line 367 also says 'In the spreadsheet, the first 58 rows provide an image, a description, and a year, to be printed on the face of the card. The next 58 rows only provide an image and a description, because the 'year' field is left empty.' 
 
Perhaps you'll be able to notice if there is something we should change to make it clear/consistent? Thank you for your patience with this!

charlottejmc replied:
I can indeed see from the spreadsheet in Figure 7 that the final image with a 'year' is at row 60, which would suggest 59 cards... I'm still about confused about how we get to that number.

copystar replied:
Thank for you for your persistence with this because you are correct. The deck is 59 cards. My apologies - I should have caught that on the first pass.

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
-This is why it is recommended to use the 'Print deck' option to make sure that all of cards are being generated properly.
-
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-8.png" alt="A screenshot of Tabletop Simulator that features an image of an octagon-shaped wooden table obscured by _Timeline_ cards that have been thrown in the air. " caption="Figure 8: Two pages generated by _nanDECK_ for printing. Note that the pages are oriented so that they can be folded together to make matched, ...
+Before you can print your deck, you must first select the _Validate Deck_ button to check that your script's syntax is valid. You can then press the _Build Deck_ button to generate your deck of cards, where you will be shown a preview of the generated cards in the right-side pane. In this display, you might see a surprising number of blank cards, but don't worry. These blank cards are inserted in ...
```
Here as well, the alt-text does not seem to match the image.

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
-nanDECK can print your deck in a multitude of ways. Each card generated by the program can be saved either as a separate graphical image or as a single PDF for printing, assembling, cutting, and gluing together. Since the common practice of game design involves multiple rounds of observed play-testing that inform iterative changes to a game's design, nanDECK's ability to easily re-generate variat ...
-
-Before you can print your deck, you first must select the 'Validate Deck' button to check the syntax of your script and ensure that it is valid. After your script has been validated, you can press the 'Build Deck' button to generate your deck of cards from the script. Finally, you can use the'Print Deck' button to have _nanDECK_ generate your deck as a PDF file for printing. 
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-8.png" alt="A screenshot of Tabletop Simulator that features an image of an octagon-shaped wooden table obscured by _Timeline_ cards that have been thrown in the air. " caption="Figure 8: Two pages generated by nanDECK for printing. Note that the pages are oriented so that they can be folded together to make matched, d ...
```
Here as well, the alt-text does not seem to match the image.

copystar replied:
Figure 8 alt-text should read:  Two pages of cards in a grid format. The right and left pages almost mirror each other but the left hand set of cards have years printed in bold and the right hand set of cards are without years printed.

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -430,99 +424,97 @@ Similar to the example that was walked through above, [`PH_nandeck_Your_Timeline
+Before we can upload our _Timeline_ deck into TTS, we will need to regenerate our cards into this required format. To do this, we have to split our original spreadsheet into two. The first spreadsheet will generate the deck's faces and, as such, will have the necessary information to generate facts, images and years. This spreadsheet will no longer need the duplicate entries without the 'years'.  ...
 
-Then it is necessary to change our script to reflect that the spreadsheets are going to be used to generate one of two files needed by TTS:
+We will have to change our script to reflect that the spreadsheets are going to be used to generate one of two files needed by TTS:
```
I wasn't exactly sure what was meant here – would you be able to rephrase this comment to clarify?

copystar replied:
Is this clearer?

"Now that we have two spreadsheets, we need to amend our previous set of instructions in the nanDECK interface in a couple of places before we can run the script."

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
````diff
@@ -430,99 +424,97 @@ Similar to the example that was walked through above, [`PH_nandeck_Your_Timeline
 ```
 
-TTS requires image files for each side of a deck to be in a standard format of page of cards that run ten in number across, and seven in number down. This standardization is necessary so the program knows how to calculate where the cards should be parsed into separate images. (If your deck has more than 70 cards, it will be necessary to parse the deck into as many as necessary under the 70 card l ...
+TTS requires the image files for each side of the deck to be provided in a certain standard format: one page containing ten cards across, and seven down. This standardization is necessary for the program to calculate where the cards should be parsed into separate images. (If your deck has more than 70 cards, it will be necessary to parse the deck into as many as necessary under the 70 card limit) ...
````
I was also a little unsure about the meaning of the section in brackets. Could you provide a slightly rephrased description here as well?

copystar replied:
Is this better?  "If your deck has more than 70 cards, then you are going to have to run through this exercise several times with generating sets of cards of 70 cards or less. For example, if you would like to generate the card faces of 200 cards, you would need to generate and process three spreadsheets: two spreadsheets of 70 rows each and one spreadsheet of 60 rows."

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -542,76 +534,71 @@ If we look at the script, [`PH_nandeck_Your_Timeline_TTS_Face.txt`](https://gith
+From this menu, you will be able to select the deck's face from your local drive. Make sure to check the box beside the option **Unique backs**. Leave the width option on 10 and the height option on 7, but do adjust the slider so that it displays the true number of cards in your deck. Once these steps are complete, hit the _Import_ button and your deck will be built for virtual play.
 
-By uploading your deck into TTS, you will be able to play your version of _Timeline_, with up to nine other invited players in an onine environment. Furthermore, if you choose to upload and make your game assets available through the [_Tabletop Simulator's Steam Workshop_](https://steamcommunity.com/app/286160/workshop/), you can add your deck to the over 11,000 sets of cards available to a commu ...
+You will now be able to play your version of _Timeline_ in an online environment with up to nine other invited players. Furthermore, if you choose to upload and make your game assets available through the [_Tabletop Simulator's Steam Workshop_](https://steamcommunity.com/app/286160/workshop/), you can add your deck to a collection of over 11,000 sets of cards that are available to a community of  ...
```
Perhaps I am missing something here, but I couldn't see where you explained how (or when) to upload the deck's back, or especially how to align it with the faces we just uploaded. I'm sorry if I'm not understanding the lesson here! I just wanted to check.

copystar replied:
This is explained in paragraph 155. If you think a figure might be useful to, I made this screenshot:

![image](https://github.com/programminghistorian/ph-submissions/assets/879434/7346414d-c7f4-4071-9b2d-1979e5677777)

charlottejmc replied:
Thanks for confirming. I think the paragraph describes it well enough without needing an image!

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
```diff
@@ -542,76 +534,71 @@ If we look at the script, [`PH_nandeck_Your_Timeline_TTS_Face.txt`](https://gith
+You will now be able to play your version of _Timeline_ in an online environment with up to nine other invited players. Furthermore, if you choose to upload and make your game assets available through the [_Tabletop Simulator's Steam Workshop_](https://steamcommunity.com/app/286160/workshop/), you can add your deck to a collection of over 11,000 sets of cards that are available to a community of  ...
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-9.png" alt="Visual description of figure image" caption="Figure 9. While one of the most delightful features of Tabletop Simulator is the ability to flip the table, the ability to throw cards around is also pretty enjoyable." %}
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-9.png" alt="Visual description of figure image" caption="Figure 9. While one of the most delightful features of Tabletop Simulator is the ability to flip the table, throwing cards around is also pretty enjoyable." %}
```
This image is missing its alt-text, but I'm pretty sure it has been left in Figure 8? In that case I imagine that some other figure is missing its alt-text!

copystar replied:
Yes, although the alt-text for Figure 9 should be amended slightly to "A screenshot of Tabletop Simulator that features an image of an octagon-shaped wooden table obscured by Timeline cards falling through the air."

charlottejmc commented:
Hi @copystar, 

Just a couple extra things which I did not tie to any specific line in the text: 

- [x] I've noticed you use both the term 'face' and 'front' of a card, in the text, in the section headings and in the code. Perhaps there is a reason for this? I wonder if it would be possible to only choose one, and apply it throughout.
- [x] I believe some steps might still be referring to files inside the [Windsor Timeline Example](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/designing-a-timeline-tabletop-simulator/Windsor%20Timeline%20Example) folder, although I can see you have removed the links to it. Could you confirm that we can delete the folder from the lesson's [/assets](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/designing-a-timeline-tabletop-simulator)?

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Fix typo
Reorder endnotes

copystar commented:
- [x] I agree it should be consistent : "face" would be the preferred term. 
- [x] Yes, you can delete the folder of Windsor Timeline Example in /assets.  
- [ ] I will review the text to see if there are any stray references to Windsor Timeline Example.  
Thank you!

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Add citation and update endnote numbers accordingly

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Image alt text and other copyedits

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Change terminology of card 'front' to 'face'

charlottejmc commented on en/drafts/originals/designing-a-timeline-tabletop-simulator.md:
````diff
@@ -64,354 +60,350 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
 
-> DUPLEX = “range front”, “range back”, number
+```
+DUPLEX = “range face”, “range back”, number
````
I've changed 'front' to 'face' here as well, because I believe "range face" is just a placeholder, and shouldn't affect the code. Do let me know if it is indeed ok this way!

charlottejmc commented:
Hi @copystar, I've updated the file with the majority of your suggestions. Thank you very much for your replies so far!

The references to the Windsor-Timeline example appear at lines:
- 283, 289, 294 (Figure 3), when you showcase the use of the `LINK` directive
- 476, 497, 502, when you split the spreadsheet into 2 for Tabletop Simulator and generate two `.png` images

I understand that we don't need to link to anything in our [/assets folder](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/designing-a-timeline-tabletop-simulator) here, because you're simply using the filenames from your own Windsor Timeline deck as examples. I wonder if it might not make things a little clearer to add small phrases like 'in our example...', or 'the spreadsheet the author used was...', so the reader knows where these filenames come from. Let me know what you think and I'd be happy to add those in if you think it's useful!

In any case, I'll delete the [Windsor Timeline Example](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/designing-a-timeline-tabletop-simulator/Windsor%20Timeline%20Example) folder from the assets. 

I also thought that it would be helpful to create a new folder called [For nanDECK](https://github.com/programminghistorian/ph-submissions/tree/copyedit-designing-a-timeline-tabletop-simulator/assets/designing-a-timeline-tabletop-simulator/For%20nanDECK) which mirrors [For Tabletop Simulator](https://github.com/programminghistorian/ph-submissions/tree/copyedit-designing-a-timeline-tabletop-simulator/assets/designing-a-timeline-tabletop-simulator/For%20Tabletop%20Simulator), in which I filed the [Build-Your-Own-Timeline.xlsx](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/designing-a-timeline-tabletop-simulator/Build-Your-Own-Timeline.xlsx), [PH_nandeck_Your_Timeline.txt](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/designing-a-timeline-tabletop-simulator/PH_nandeck_Your_Timeline.txt), [Your-Timeline-Duplex-Printing-Result.pdf](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/designing-a-timeline-tabletop-simulator/Your-Timeline-Duplex-Printing-Result.pdf) and 6 `.png` assets used in the nanDECK section of the lesson. I can edit Line 399 slightly to explain the new structure of the assets folder.

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

additional edits

charlottejmc committed:
Delete assets/designing-a-timeline-tabletop-simulator/Windsor Timeline Example directory

delete unused asset

charlottejmc committed:
Delete assets/designing-a-timeline-tabletop-simulator directory

delete to reupload

charlottejmc committed:
Reupload files in two sub folders

copystar commented:
As I was re-reading the draft, I noticed the following

- [x] There is a premature paragraph break at the end of ¶5
- [x] "lession" in ¶8 should be "lesson"
- [x] also in ¶8, "a PDF" should read, "pngs"
- [x] ¶45:  correction: a set of 59 cards (not 58)
- [x] is there a way to scale down the image of Figure 6: it looks very large to my eyes can probably could be scaled down by half

copystar commented:
> Hi @copystar, I've updated the file with the majority of your suggestions. Thank you very much for your replies so far!
> 
> The references to the Windsor-Timeline example appear at lines:
> 
>     * 283, 289, 294 (Figure 3), when you showcase the use of the `LINK` directive
> 
>     * 476, 497, 502, when you split the spreadsheet into 2 for Tabletop Simulator and generate two `.png` images
> 
> 
> I understand that we don't need to link to anything in our [/assets folder](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/designing-a-timeline-tabletop-simulator) here, because you're simply using the filenames from your own Windsor Timeline deck as examples. I wonder if it might not make things a little clearer to add small phrases like 'in our example...', or 'the spreadsheet the author used was...', so the reader knows where these filenames come from. Let me know what you think and I'd be happy to add those in if you think it's useful!
> 
> In any case, I'll delete the [Windsor Timeline Example](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/designing-a-timeline-tabletop-simulator/Windsor%20Timeline%20Example) folder from the assets.
> 
> I also thought that it would be helpful to create a new folder called [For nanDECK](https://github.com/programminghistorian/ph-submissions/tree/copyedit-designing-a-timeline-tabletop-simulator/assets/designing-a-timeline-tabletop-simulator/For%20nanDECK) which mirrors [For Tabletop Simulator](https://github.com/programminghistorian/ph-submissions/tree/copyedit-designing-a-timeline-tabletop-simulator/assets/designing-a-timeline-tabletop-simulator/For%20Tabletop%20Simulator), in which I filed the [Build-Your-Own-Timeline.xlsx](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/designing-a-timeline-tabletop-simulator/Build-Your-Own-Timeline.xlsx), [PH_nandeck_Your_Timeline.txt](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/designing-a-timeline-tabletop-simulator/PH_nandeck_Your_Timeline.txt), [Your-Timeline-Duplex-Printing-Result.pdf](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/designing-a-timeline-tabletop-simulator/Your-Timeline-Duplex-Printing-Result.pdf) and 6 `.png` assets used in the nanDECK section of the lesson. I can edit Line 399 slightly to explain the new structure of the assets folder.

I think the suggestion to use phrasing like ' 'the spreadsheet the author used was..' will be useful, thank you. I also like the suggestion to mirror the folders so that they read similarly, "For..."

charlottejmc commented:
> As I was re-reading the draft, I noticed the following
> 
> * [x]  There is a premature paragraph break at the end of ¶5
> * [x]  "lession" in ¶8 should be "lesson"
> * [x]  also in ¶8, "a PDF" should read, "pngs"
> * [x]  ¶45:  correction: a set of 59 cards (not 58)
> * [x]  is there a way to scale down the image of Figure 6: it looks very large to my eyes can probably could be scaled down by half

Hi @copystar,

Thanks for these. I understand you were checking the [the preview](https://programminghistorian.github.io/ph-submissions/en/drafts/originals/designing-a-timeline-tabletop-simulator) here. I'm sorry that I didn't make this clear, but the preview is generated from the [draft which is currently on the main gh-pages branch](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/en/drafts/originals/designing-a-timeline-tabletop-simulator.md). My copyedits won't reflect in the preview until I merge this branch! You can check the most up-to-date version of the copyedited draft by clicking on [Files changed](https://github.com/programminghistorian/ph-submissions/pull/600/files) of this Pull Request and then `en/drafts/originals` > [designing-a-timeline-tabletop-simulator](https://github.com/programminghistorian/ph-submissions/pull/600/files#diff-07cecdc6f8e6aee9df05aed15b61f40ea654b270224ab7f3c959d59482e8aa0d).

- [ ] Issues 1, 2, and 3 you've flagged above have already been fixed in my edits.
- [ ] Thank you for noting this. I've actually now changed every instance of `58` to `59`, and hope that's correct. 
- [ ] Yes, the images are very big right now! Part of the typesetting process (which will come after I merge the copyedits) will involve sizing the images down to a consistent standard.
- [ ] I've also now made the changes you approved above (create a folder `For nanDECK` and using phrasing like  'the spreadsheet the author used was...'
- [ ] I've changed the title to your preferred version: "Designing a deck of _Timeline_ cards for tabletops and Tabletop Simulator"

I'm happy to merge my edits at this stage, and move on to the typesetting phase. We'll still be able to make changes (if you find any further issues) during that phase.

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Additional edits to resolve comments

charlottejmc committed:
Update designing-a-timeline-tabletop-simulator.md

Minor edit

copystar commented:
Thank you!

copystar commented:
Thank you for everyone who made this lesson available to the world! 

I'm going to write a blog post (and make a special law-related game for it) shortly to help share this work more widely.

I did notice that the DOI doesn't seem to resolve yet. Can this be submitted so I can add this work to my ORCiD profile?

Thank you all again, Mita
